### ISSUE TITLE
`urllib3` pinned version causes issues

### ISSUE THREAD

**Issue by tbrlpld** [CyberSource/cybersource-rest-client-python#86]
I am wondering why the version of `urllib3` is pinned in the [`requirements.txt`](https://github.com/CyberSource/cybersource-rest-client-python/blob/master/requirements.txt#L8)? 

It's currently pinned to `1.26.5`, but used to be pinned to other versions. 

This pinning causes compatibility issues for me. 

E.g. when trying to install this side-by-side with Celery, I get this issue from Poetry:

```
...
  And because celery (5.2.1) depends on both kombu (>=5.2.2,<6.0) and kombu (*), celery (5.2.1) requires urllib3 (<1.26 || >=1.26.7) or pycurl (7.43.0.2).
  And because cybersource-rest-client-python (0.0.36) depends on urllib3 (1.26.5)
   and myproject depends on pycurl (7.44.1), celery (5.2.1) is incompatible with cybersource-rest-client-python (0.0.36).
  So, because myproject depends on both cybersource-rest-client-python (0.0.36) and celery (5.2.1), version solving failed.

```

It is very uncommon for librarys like `cybersource-rest-client-python` to pin versions. Normally versions should only be pinned in applications. Library should only exclude version which cause known issues. Everything else should be left to the using application. 

So, is there a specific reason for the pin? Or can this be removed or at least turned into something more flexible?

**Comment by tbrlpld** [comment]
cc @gnongsie

**Comment by JuanPabloRN30** [comment]
I'm having the same problem here. Your comment was a while ago, How did you handle it? @tbrlpld

**Comment by tbrlpld** [comment]
> I'm having the same problem here. Your comment was a while ago, How did you handle it? @tbrlpld 

Sorry @JuanPabloRN30 it's been a while and I don't recall how we got around this

**Comment by gaubansa** [comment]
It's been already fixed in Cybersource Python SDK v0.0.45 onwards.

**Comment by tbrlpld** [comment]
https://github.com/CyberSource/cybersource-rest-client-python/releases/tag/0.0.45
